packages/rulesets/src/native/utilities/arm-helper.ts

```diff
@@ -560,6 +560,10 @@ export class ArmHelper {
     return resourceCollectMap
   }
 
+  public isPathTenantLevel(path: string) {
+    return path.match(this.OperationApiRegEx)
```

Comment:
Since OperationApiRegEx is `"^/providers/[^/]+/operations$"` this will only match /operations paths. Do we want to match other tenant-level paths?

Reply:
I assumed this was the only check needed, thanks Brett.